src/imitation/util/logger.py

```diff
@@ -170,30 +170,30 @@ def configure(folder: str, format_strs: Optional[Sequence[str]] = None) -> None:
     assert is_configured()
 
 
-def logkv(key, val) -> None:
-    """Alias for `stable_baselines.logger.logkv`."""
-    sb_logger.logkv(key, val)
+def record(key, val, exclude=None) -> None:
+    """Alias for `stable_baselines3.logger.record`."""
+    sb_logger.record(key, val, exclude)
 
 
-def dumpkvs() -> None:
-    """Alias for `stable_baselines.logger.dumpkvs`."""
-    sb_logger.dumpkvs()
+def dump(step=0) -> None:
```

Comment:
What happens if I don't provide the step? Does it do anything bad?

Reply:
this is only useful for tensorboard

Reply:
Thanks for clarifying!

Reply:
Yes, we do. `get_log_dict()` feels a bit too magic to me, though, given that different algorithms use different keys to track training progress. I've updated BC and the adversarial algorithms to explicitly pass a step instead.